tests/Toolkit/StrTest.php

```diff
@@ -8,50 +8,50 @@ class StrTest extends TestCase
 {
     public function testAscii()
     {
-        $this->assertEquals('aouss', Str::ascii('äöüß'));
-        $this->assertEquals('Istanbul', Str::ascii('İstanbul'));
-        $this->assertEquals('istanbul', Str::ascii('i̇stanbul'));
-        $this->assertEquals('Nashata istorija', Str::ascii('Нашата история'));
+        $this->assertSame('aouss', Str::ascii('äöüß'));
+        $this->assertSame('Istanbul', Str::ascii('İstanbul'));
+        $this->assertSame('istanbul', Str::ascii('i̇stanbul'));
+        $this->assertSame('Nashata istorija', Str::ascii('Нашата история'));
     }
 
     public function testAfter()
     {
         $string = 'Hellö Wörld';
 
         // case sensitive
-        $this->assertEquals(' Wörld', Str::after($string, 'ö'));
-        $this->assertEquals(false, Str::after($string, 'Ö'));
-        $this->assertEquals(false, Str::after($string, 'x'));
+        $this->assertSame(' Wörld', Str::after($string, 'ö'));
+        $this->assertSame('', Str::after($string, 'Ö'));
+        $this->assertSame('', Str::after($string, 'x'));
```

Comment:
I wonder why `false` worked before. Was it only because `assertEquals()` wasn't strict enough or did the methods actually return `false`.

Reply:
As far as I can tell, assertEqual just was fine with it. The return type hinting turned the Boolean into an empty string.

Reply:
👌